### PR TITLE
add wiw badges to sismo playground

### PR DESCRIPTION
**Pull request by KevinYum**
Add two badges to playground:
- wiw-crypto-memento
- wiw-nft-degen-traders

Use snapshotted address list as group for now, will switch to use WIW data provider after it is ready!

### CODE REVIEW

File: group-generators/generators/wiw-crypto-memento/index.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// sismo-hub/group-generators/generators/wiw-crypto-memento/index.ts
```
Comment (yum0e) [inline group-generators/generators/wiw-crypto-memento/index.ts:1]: remove the commentary from the tutorial (same remark for the other group generator)

File: group-generators/generators/wiw-crypto-memento/index.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    GroupWithData,
+    AccountSource,
+} from "topics/group";
+import {
+    GenerationContext,
+    GenerationFrequency,
+    GroupGenerator,
+} from "topics/group-generator";
+
+// Here you are hacker 😈
```
Comment (yum0e) [inline group-generators/generators/wiw-crypto-memento/index.ts:17]: remove the commentary from the tutorial (same remark for the other group generator)

File: .gitignore [hunk 3]
```diff
@@ -5,4 +5,5 @@
 coverage
 .env*
 /log
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea
```
Comment (yum0e) [inline .gitignore:9]: Can you remove .idea please?

**Comment by yum0e** [comment]
Hey @KevinYum! Awesome PR 😁 I left some minor commentaries (useless comments from tuto and a strange `.idea` in the .gitignore. Feel free to ping when it is resolved.

File: .gitignore (hunk 3 above)
Comment (KevinYum) [reply .gitignore]: Done.

File: group-generators/generators/wiw-crypto-memento/index.ts (hunk 2 above)
Comment (KevinYum) [reply group-generators/generators/wiw-crypto-memento/index.ts]: Done.

File: group-generators/generators/wiw-crypto-memento/index.ts (hunk 1 above)
Comment (KevinYum) [reply group-generators/generators/wiw-crypto-memento/index.ts]: Done.

**Comment by KevinYum** [comment]
Thanks for the comment :)
All fixed.

**Comment by KevinYum** [comment]
Per discussion in telegram, we will deprecate this PR and create a smaller group

**Comment by yum0e** [comment]
Hey! Thanks for the changes. You can totally keep this PR and add your new groups to it, no need to deprecate I think :)

**Comment by KevinYum** [comment]
> Hey! Thanks for the changes. You can totally keep this PR and add your new groups to it, no need to deprecate I think :)

Haha what a coincidence. I've just updated this PR and found your comment here :)
Could you help check it again @yum0e

**Review by leosayous21 — APPROVED** [review]

**Pull request merged**